### DISCUSSION TITLE
[クルー/Crewmate] : ティーチャー/Teacher (能力を教える)

### DISCUSSION THREAD

**Discussion by yukieiji** [yukieiji/ExtremeRoles#323] [Ideas]
- アイディアが完全に固まりきってないので意見求めも含めてディスカッション化
  - これらの能力、仕様は変更されることがあります

## 能力詳細
- 名前：ティーチャー/Teacher
- 色：
- 陣営：クルー/Crewmate

- 役職コンセプト：他人に能力を教授する
  - すべての陣営に対して有利な能力を付与できるようにする

- ボタン能力：「？？？」
  - プレイヤーを対象にして発動
  - 発動が完了するとそのプレイヤーに追加の役職を付与する(サーヴァントと同じ様な感じ)

## 未検討事項
- 何の役職追加しようね？

**Comment by flog4624** [comment]
他人に役職を与える、ですか。面白そうですね
与える役職として自分が思い浮かんだのは
・スーパーバイザーやモデレーターなど、既存役職の能力をそのまま付与。この場合、オプションで付与する役職を決めておく必要がある
・上記の亜種で、ティーチャーの能力用に弱体化調整された既存役職の能力を付与。例えばフェンサーの場合はキル能力をガードするだけでカウンターはできない
・ティーチャーの能力専用のオリジナル能力を付与。例えば「自分がアドミンを使う場合のみ、クルーのアイコンに色がつく」
の3択ですね。
しかし、元々特殊能力がある役職に能力を使ったら能力ボタンが2つ以上になりますね…

**Reply by yukieiji** [reply]
> ・スーパーバイザーやモデレーターなど、既存役職の能力をそのまま付与。この場合、オプションで付与する役職を決めておく必要がある
・上記の亜種で、ティーチャーの能力用に弱体化調整された既存役職の能力を付与。例えばフェンサーの場合はキル能力をガードするだけでカウンターはできない

前にもあったのですが基本的にこういう役職は別の役職を付与しないとオプションを設定をどうするのかとかクルー役職が増えたらどうするのかとか考える必要があって結果的にめんどくなります

> しかし、元々特殊能力がある役職に能力を使ったら能力ボタンが2つ以上になりますね…

それはもう仕方ないでしょう、サーヴァントも一緒です
